terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,72 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                 = "search-service" # this must match the stack name the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "sic-code-api"
+  container_port             = 8080
+  eric_port                  = "10000"
+  docker_repo                = "sic-code-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 21
```

Comment:
The load balancer priority will need a different value to other applications in the stack that expose an end point. This should be determined by the architect before we merge this.

Reply:
Checked the lbs and lb priority in the "search" stack and 21 is free on the load balancer that the sic-code-api will be using. Spoke to toba from platform and is happy with 21 in that case